.github/workflows/pypi_publish.yml

```
@@ -59,3 +129,209 @@
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/setup-python@v4
+        with:
+          python-version: "3.10"
+      - name: Checkout Airbyte Platform Internal
+        uses: actions/checkout@v3
+        with:
+          repository: airbytehq/airbyte-platform-internal
+          token: ${{ secrets.GH_PAT_MAINTENANCE_OCTAVIA }}
+      - name: Update Builder's CDK version to ${{ env.VERSION }}
+        run: |
```

actionlint (1.7.4)

shellcheck reported issue in this script: SC2086:info:5:6: Double quote to prevent globbing and word splitting

(shellcheck)